subsys/fs/nffs_fs.c

```diff
@@ -524,6 +524,17 @@ static int nffs_rename(struct fs_mount_t *mountp, const char *from,
 	return translate_error(rc);
 }
 
+static bool get_flash_size_cb(const struct flash_pages_info *info, void *data)
+{
+	struct nffs_flash_desc *flash_desc = data;
+
+	if (flash_desc->area_offset + flash_desc->area_size == info->start_offset) {
+		flash_desc->area_size += info->size;
```

Comment:
why not '=' instead?

After this line;
this condition should be valid exactly for the one flash sector - so use `return false' for  finish the iteration once this happened,

Reply:
one? why not all?

Reply:
Here is the use of all the remaining pages, not just one page.